--- config.def.h	2019-02-17 14:46:25.200546884 +0000
+++ config.h	2019-04-20 20:56:16.863894588 +0100
@@ -1,11 +1,13 @@
 /* See LICENSE file for copyright and license details. */
 
+// TODO: configure better/sane zoom keys
+
 /*
  * appearance
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Inconsolata:pixelsize=18:antialias=true";
 static int borderpx = 2;
 
 /*
@@ -49,7 +51,7 @@ static unsigned int actionfps = 30;
  * blinking timeout (set to 0 to disable blinking) for the terminal blinking
  * attribute.
  */
-static unsigned int blinktimeout = 800;
+static unsigned int blinktimeout = 0;
 
 /*
  * thickness of underline and bar cursors
@@ -84,25 +86,37 @@ unsigned int tabspaces = 8;
 
 /* Terminal colors (16 first used in escape sequence) */
 static const char *colorname[] = {
+
 	/* 8 normal colors */
-	"black",
-	"red3",
-	"green3",
-	"yellow3",
-	"blue2",
-	"magenta3",
-	"cyan3",
-	"gray90",
+	"#262626",  // black
+	"#cf3f61",  // red
+	"#7bb75b",  // green
+	"#e9b32a",  // yellow
+	"#3465a4",  // blue
+	"#a57fc4",  // magenta
+	"#389aad",  // cyan
+	"#fafaf6",  // white
+
+	/* /1* Tango *1/ */
+	/* /1* 8 normal colors *1/ */
+	/* "#262626",  // black */
+	/* "#cc0000",  // red */
+	/* "#4e9a06",  // green */
+	/* "#c4a000",  // yellow */
+	/* "#3465a4",  // blue */
+	/* "#75507b",  // magenta */
+	/* "#06989a",  // cyan */
+	/* "#d3d7cf",  // white */
 
 	/* 8 bright colors */
-	"gray50",
-	"red",
-	"green",
-	"yellow",
-	"#5c5cff",
-	"magenta",
-	"cyan",
-	"white",
+	"#555753",  // black
+	"#ef2929",  // red
+	"#8ae234",  // green
+	"#fce94f",  // yellow
+	"#729fcf",  // blue
+	"#ad7fa8",  // magenta
+	"#34e2e2",  // cyan
+	"#eeeeee",  // white
 
 	[255] = 0,
 
@@ -116,7 +130,7 @@ static const char *colorname[] = {
  * Default colors (colorname index)
  * foreground, background, cursor, reverse cursor
  */
-unsigned int defaultfg = 7;
+unsigned int defaultfg = 15;
 unsigned int defaultbg = 0;
 static unsigned int defaultcs = 256;
 static unsigned int defaultrcs = 257;
