--- config.def.h	2018-05-29 23:03:59.912247435 +0100
+++ config.h	2018-06-03 10:50:02.905391948 +0100
@@ -5,7 +5,7 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "Liberation Mono:pixelsize=12:antialias=true:autohint=true";
+static char *font = "Tamsyn:pixelsize=16";
 static int borderpx = 2;
 
 /*
@@ -49,7 +49,7 @@
  * blinking timeout (set to 0 to disable blinking) for the terminal blinking
  * attribute.
  */
-static unsigned int blinktimeout = 800;
+static unsigned int blinktimeout = 0;
 
 /*
  * thickness of underline and bar cursors
