--- config.def.h	2018-05-30 20:47:12.052807247 +0100
+++ config.h	2018-06-03 15:43:28.956874898 +0100
@@ -5,8 +5,8 @@
 static const unsigned int snap      = 32;       /* snap pixel */
 static const int showbar            = 1;        /* 0 means no bar */
 static const int topbar             = 1;        /* 0 means bottom bar */
-static const char *fonts[]          = { "monospace:size=10" };
-static const char dmenufont[]       = "monospace:size=10";
+static const char *fonts[]          = { "Tamsyn:size=16" };
+static const char dmenufont[]       = "Tamsyn:size=16";
 static const char col_gray1[]       = "#222222";
 static const char col_gray2[]       = "#444444";
 static const char col_gray3[]       = "#bbbbbb";
@@ -44,6 +44,11 @@
 };
 
 /* key definitions */
+#define XF86MonBrightnessDown           0x1008ff03
+#define XF86MonBrightnessUp             0x1008ff02
+#define XF86AudioMute                   0x1008ff12
+#define XF86AudioLowerVolume            0x1008ff11
+#define XF86AudioRaiseVolume            0x1008ff13
 #define MODKEY Mod1Mask
 #define TAGKEYS(KEY,TAG) \
 	{ MODKEY,                       KEY,      view,           {.ui = 1 << TAG} }, \
@@ -57,7 +62,12 @@
 /* commands */
 static char dmenumon[2] = "0"; /* component of dmenucmd, manipulated in spawn() */
 static const char *dmenucmd[] = { "dmenu_run", "-m", dmenumon, "-fn", dmenufont, "-nb", col_gray1, "-nf", col_gray3, "-sb", col_cyan, "-sf", col_gray4, NULL };
-static const char *termcmd[]  = { "st", NULL };
+static const char *termcmd[]  = { "st", "-e", "tmux", NULL };
+static const char *cmdbrightnessup[]    = { "sudo", "$HOME/bin/bright", "up",   NULL };
+static const char *cmdbrightnessdown[]  = { "sudo", "$HOME/bin/bright", "down", NULL };
+static const char *cmdsoundup[]     = { "audio", "-u", "5", NULL };
+static const char *cmdsounddown[]   = { "audio", "-d", "5", NULL };
+static const char *cmdsoundtoggle[] = { "audio", "-m",      NULL };
 
 static Key keys[] = {
 	/* modifier                     key        function        argument */
@@ -94,6 +104,11 @@
 	TAGKEYS(                        XK_8,                      7)
 	TAGKEYS(                        XK_9,                      8)
 	{ MODKEY|ShiftMask,             XK_q,      quit,           {0} },
+	{ 0,                            XF86MonBrightnessDown,     spawn,         {.v = cmdbrightnessdown } },
+	{ 0,                            XF86MonBrightnessUp,       spawn,         {.v = cmdbrightnessup } },
+	{ 0,                            XF86AudioMute,             spawn,         {.v = cmdsoundtoggle } },
+	{ 0,                            XF86AudioRaiseVolume,      spawn,         {.v = cmdsoundup } },
+	{ 0,                            XF86AudioLowerVolume,      spawn,         {.v = cmdsounddown } },
 };
 
 /* button definitions */
